src/router/client.js

```diff
@@ -80,6 +84,26 @@ const router = new VueRouter({
   },
 });
 
+// eslint-disable-next-line
```

Comment:
Disabling ESLint is discouraged. Please remove this

Reply:
Done